Content.Server/GlobalVerbs/RejuvenateVerb.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Content.Server.GameObjects;
+using Content.Server.GameObjects.Components.Nutrition;
+using Content.Shared.GameObjects;
+using Robust.Server.Console;
+using Robust.Server.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+
+namespace Content.Server.GlobalVerbs
+{
+    /// <summary>
+    ///     Completely removes all damage from the DamageableComponent (heals the mob).
+    /// </summary>
+    [GlobalVerb]
+    class RejuvenateVerb : GlobalVerb
+    {
+        public override string GetText(IEntity user, IEntity target) => "Rejuvenate";
+        public override bool RequireInteractionRange => false;
+
+        public override VerbVisibility GetVisibility(IEntity user, IEntity target)
+        {
+            var groupController = IoCManager.Resolve<IConGroupController>();
+
+            if (user.TryGetComponent<IActorComponent>(out var player))
```

Comment:
Could you make it so this verb only shows up if the entity in question has some kind of component that can actually be rejuvenated?

So basically return `VerbVisiblity.Invisible` here if the entity has none of `DamageableComponent`, `HungerComponent` or `ThirstComponent`